src/containers/PackingOverview/RemindersPage/RemindersPage.js

```diff
@@ -17,60 +19,139 @@ const RemindersPage = props => {
     bag_id
   } = props;
 
-  const [todoList, setTodoList] = useState(null);
+  const [todoList, setTodoList] = useState([]);
+  const [todoListId, setTodoListId] = useState(null);
+  const [shoppingListId, setShoppingListId] = useState(null);
   const [shoppingList, setShoppingList] = useState([]);
   const [alertDisplay, setAlertDisplay] = useState(false);
   const [itemInput, setItemInput] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [todoDisplay, setTodoDisplay] = useState(false);
+  const [todoInput, setTodoInput] = useState("");
 
   useEffect(() => {
-    console.log("lists", lists);
+    console.log(lists, "lists");
+    console.log("selected", selectedList);
     if (!lists.length) return;
     for (let list of lists) {
       getList(list);
     }
   }, [lists]);
 
-  const createList = () => {
-    if (lists.list_type === "Todo List" || "Shopping List") {
-      setAlertDisplay(true);
-    } else {
-      axios({
-        method: "post",
-        url: "http://localhost:5000/todolist/",
-        data: {
-          name: "",
-          trip_id,
-          list_type: selectedList
-        }
+  useEffect(() => {
+    console.log(shoppingList, "s list");
+    console.log(shoppingListId, "s id");
+    console.log(todoList, "tdl");
+    console.log(todoListId, "tdl id");
+  }, []);
+
+  useEffect(() => {
+    axios({
+      method: "get",
+      url: BASE_URL + "/todolist/todo/" + lists.todolist_id
+    })
+      .then(data => {
+        console.log(data);
+        setTodos(data);
       })
-        .then(res => {
-          console.log("list created");
-          updateLists();
-        })
-        .catch(err => {
-          console.log(err);
-        });
+      .catch(err => {
+        console.log(err);
+      });
+  }, [todoList, shoppingList]);
+
+  const createList = () => {
+    for (let i = 0; i < lists.length; i++) {
+      if (lists[i].list_type === selectedList) {
+        setAlertDisplay(true);
+        return;
+      }
     }
+    axios({
+      method: "post",
+      url: "http://localhost:5000/todolist/",
```

Comment:
Why don't you use the backendUrlConnect in services?

Reply:
This is totally my fault, I forgot about the backendUrlConnect file when we worked on this section. 😅

Reply:
Whoops, that's right, I forgot. I fixed it 👍